Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -3582,10 +3580,12 @@ void UCesiumGltfComponent::UpdateFade(float fadePercentage, bool fadingIn) {
     return;
   }
 
+  fadePercentage = glm::clamp(fadePercentage, 0.0f, 1.0f);
```

Comment:
Did moving this line fix anything? Or was it just code cleanup?

Reply:
Just code cleanup from when I was debugging the fades (which was ultimately not necessary)